src/main/resources/spring/query-context.xml

```diff
@@ -77,4 +76,28 @@
         <constructor-arg value="select id, name from country;"/>
     </bean>
 
+    <bean id="insertMovie" class="java.lang.String">
+        <constructor-arg value="insert into movie (name_russian, name_native, year_of_release, description, rating, price, picture_path) values (:nameRussian, :nameNative, :yearOfRelease, :description, :rating, :price, :picturePath);"/>
+    </bean>
+
+    <bean id="insertMovieCountry" class="java.lang.String">
+        <constructor-arg value="insert into movie_country(movie_id, country_id) values(:movieId, :countryId);"/>
+    </bean>
+
+    <bean id="insertMovieGenre" class="java.lang.String">
+        <constructor-arg value="insert into movie_genre (movie_id, genre_id) values (:movieId, :genreId);"/>
+    </bean>
+
+    <bean id="updateMovie" class="java.lang.String">
+        <constructor-arg value="update movie set name_russian =:nameRussian, name_native =:nameNative, year_of_release =:yearOfRelease, description=:description, rating=:rating, price=:price, picture_path=:picturePath where id =:movieId;"/>
+    </bean>
```

Comment:
According to business requirements for movie update can be provided not complete set of columns. Would be good if you will consider this in the update statement, otherwise some values can be overwritten by NULLs

Reply:
I've got requirement if we get NULL for any attribute then we set it to NULL.